### ISSUE TITLE
Any examples in the context of adaptive design where one of pinball loss or interval score would be preferred over the other?

### ISSUE THREAD

**Issue by sgbaird** [uncertainty-toolbox/uncertainty-toolbox#62]
Having trouble deciding which one is more relevant to my use-case. I'm leaning towards interval score #57, but if you had an example or two of when one might be preferred over the other, I think this would help it be a lot more concrete for me. Any that come to mind and comments on which would be preferred? For example:
- tuning the hyperparameters of a neural network to maximize regression accuracy
- suggesting the next experiment to run to find new, superhard materials

**Comment by IanChar** [comment]
Hi @sgbaird, perhaps @YoungseogChung would have better advice, but I think a good scenario to use the interval score in would be if you actually want to form centered intervals in your use of the uncertainty model e.g. along with the mean predict a lower and upper bound on what the value could be.
